dev/deps/spark-deps-hadoop-2.7

```diff
@@ -125,7 +125,6 @@ jetty-6.1.26.jar
 jetty-sslengine-6.1.26.jar
 jetty-util-6.1.26.jar
 jline-2.14.6.jar
-joda-time-2.10.5.jar
```

Comment:
Instead of this, `joda-time-2.9.9.jar` seems to occur transitively. Could you check the dependency tree and remove that, too?
```
+++ b/dev/pr-deps/spark-deps-hadoop-2.7
@@ -125,6 +125,7 @@ jetty-6.1.26.jar
 jetty-sslengine-6.1.26.jar
 jetty-util-6.1.26.jar
 jline-2.14.6.jar
+joda-time-2.9.9.jar
 jodd-core-3.5.2.jar
 jpam-1.1.jar
 json4s-ast_2.12-3.6.6.jar
```